code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -77,17 +77,17 @@
 	growthstages = 3
 	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
 	growing_icon = 'icons/obj/hydroponics/growing_mushrooms.dmi'
-	reagents_add = list("psilocybin" = 0.04, "nutriment" = 0, "amanitin" = 0.3)
+	reagents_add = list("psilocybin" = 0.04, "nutriment" = 0, "nothing" = 0.2, "phalloidin" = 0.2)
 	rarity = 30
-	origin_tech = "biotech=5"
 
 /obj/item/food/snacks/grown/mushroom/angel
 	seed = /obj/item/seeds/angel
 	name = "destroying angel"
-	desc = "<I>Amanita Virosa</I>: Deadly poisonous basidiomycete fungus filled with alpha amanitin."
+	desc = "<I>Amanita Virosa</I>: Deadly poisonous basidiomycete fungus filled with phalloidin."
 	icon_state = "angel"
 	filling_color = "#C0C0C0"
 	tastes = list("destroying angel" = 1)
+	origin_tech = "biotech=6" // as useful as ambrosia gaia, and quicker to get - but you have to leave traitor tools lying around...
```

Comment:
Not sure I understand the comment, traitor tools? I don't think a single destroying angel will useful enough for a traitor. Perhaps single syringe or pill. Or poison. But they have access to other tools easily enough.

Reply:
With a small balance tweak perhaps? Botany doesn't need more killing tools.